### ISSUE TITLE
Update dymint version

### ISSUE THREAD

**Issue by ItayLevyOfficial** [dymensionxyz/rollapp#44]

**Comment by ItayLevyOfficial** [comment]
Can't compile after updating dymint, fails with
```
itaylevy@Itays-MacBook-Pro rollapp % make build
go build  -o build/rollappd -ldflags '-X github.com/cosmos/cosmos-sdk/version.Name=dymension-rdk -X github.com/cosmos/cosmos-sdk/version.AppName=rollappd -X github.com/cosmos/cosmos-sdk/version.Version=v0.1.0-rc4-2-g23cd1b8 -X github.com/cosmos/cosmos-sdk/version.Commit=23cd1b813c91fadebf2863d93057bbff600f85b7 -X github.com/tendermint/tendermint/version.TMCoreSemVer=v0.34.28' ./cmd/rollappd
# github.com/cosmos/gogoproto/proto
../../go/pkg/mod/github.com/cosmos/gogoproto@v1.4.8/proto/merge.go:286:25: type func(x *descriptorpb.FileDescriptorProto, y *descriptorpb.FileDescriptorProto) bool of func(x, y *descriptorpb.FileDescriptorProto) bool {…} does not match inferred type func(a *descriptorpb.FileDescriptorProto, b *descriptorpb.FileDescriptorProto) int for func(a E, b E) int
make: *** [build] Error 2
itaylevy@Itays-MacBook-Pro rollapp % 
```

**Comment by omritoptix** [comment]
yea it's familiar. happens because of some packages in dymint go.mod. the fix was applied directly into release/v0.5.x [here](https://github.com/dymensionxyz/dymint/pull/453) but should also be applied to main.

**Comment by ItayLevyOfficial** [comment]
solved it already
